bot/cogs/fun.py

```diff
@@ -94,6 +95,30 @@ def __init__(self, bot: Bot):
         if "revive" in message.content.lower():
             await emojify(message, "nou")
 
+        # Ask if user has contacted support before letting them ping staff
+        if self.staff_role is None:
+            for y in self.bot.get_server(SERVER_ID).roles:
+                if y.id == STAFF_ROLE_ID:
+                    self.staff_role = y
+                break
+
+        if FAKE_STAFF_ROLE_ID in message.raw_role_mentions and message.author.id != self.bot.user.id:
+            msg = await self.bot.send_message(message.channel,
+                                              'Have you tried emailing support@joincyberdiscovery.com? Please make '
+                                              'sure to do that before unnecessarily pinging the staff!')
```

Comment:
I'm not sure I agree with this wording (but I might be wrong about how it will be read!). It should more directly map with an action a bot will do, else the question will be read as rhetorical, and the up/down mapping may become reversed to the user ("I did contact support, hence, this is unhelpful, so I'll downvote")

I propose "Continue with the mention anyway?" and make the "Have you ... support?" question more authoritative, e.g. "Contact support before mentioning them here"